pilot/pkg/networking/v1alpha3/config.go

```diff
@@ -734,20 +737,33 @@ func buildInboundListeners(mesh *meshconfig.MeshConfig, node model.Proxy,
 // the pod.
 // So, if a user wants to use kubernetes probes with Istio, she should ensure
 // that the health check ports are distinct from the service ports.
-func buildMgmtPortListeners(mesh *meshconfig.MeshConfig, managementPorts model.PortList,
-	managementIP string) []*xdsapi.Listener {
+func buildMgmtPortListeners(managementPorts model.PortList, managementIP string) []*xdsapi.Listener {
 	listeners := make([]*xdsapi.Listener, 0, len(managementPorts))
 
+	if managementIP == "" {
+		managementIP = "127.0.0.1"
```

Comment:
define constant. i also wonder if having this sort of internal behavior is a good idea - at least it should be spelled out in the function definition.

Reply:
I think this got replaced by Shriram code - but technically the management ports should forward
 to localhost, not sure why it was trying to use the interface IP address.